contrib/deploy/README.md

```diff
@@ -0,0 +1,80 @@
+# Honeypot Deployment on Fly.io
+
+This repository provides sample `fly.toml` files for deploying a Honeypot validator node to [Fly.io](https://fly.io).  
+It contains two preconfigured examples for deployment to **Ethereum Mainnet** and **Sepolia testnet**.
+
+> **Note:** The `fly.toml` files rely on the Dockerfile located at `../compose/Dockerfile`.  
+> Ensure you place or run these configs from the correct directory.
+
+## 🛠️ Requirements
+
+Before proceeding, ensure you have the following:
+
+- A valid **Fly.io account** ([sign up](https://fly.io/docs/hands-on/sign-up/))
+- An **RPC URL** for the target network:
+  - ⚠️ *Free Alchemy plans* may hit a 500-block range limit, causing sync issues.
+  - ✅ Prefer [Infura](https://infura.io) (even free tier supports limited usage) or upgrade to a paid RPC provider.
+- A **private key** funded with ETH for transaction fees on your selected network (Mainnet or Sepolia).
+
+## 🚀 Deployment Steps
+
+### 1. Install the Fly.io CLI
+
+Choose the installation method for your OS:
+
+```bash
+brew install flyctl                    # macOS (Homebrew)
+curl -L https://fly.io/install.sh | sh # Linux/macOS
+pwsh -Command "iwr https://fly.io/install.ps1 -useb | iex"  # Windows (PowerShell)
+```
+
+### 2. Authenticate with Fly.io
+
+Authenticate your CLI using the below command:
+
+```shell
+fly auth login
+```
+
+### 3. Choose Your Network Configuration
+
+Rename the fly.toml file appropriately: If you intend to deploy a sepolia or mainnet node then you need to rename the appropriate fly.toml.example file to fly.toml. 
+
+```shell
+# For Mainnet
+mv fly.toml.mainnet.example fly.toml
+
+# Or for Sepolia
+mv fly.toml.sepolia.example fly.toml
+```
+
+### 4. Set Your App Name
+
+Next replace the `<Application_name>` in your fly.toml file with your desired application name.
+
+### 5. Create the Fly.io App
+
+Create an application on fly.io using the exact name you used above:
+
+```shell
+fly app create <Application_name>
+```
```

Comment:
I'm not sure the `fly app create` is necessary.

If the file is there and the app is not created yet, you can `fly launch --config=`.

So the template doesn't need to have the app = '<Application Name>` and it will be generated.